### PR TITLE
feat: opengraph user profiles

### PR DESCRIPTION
**Pull request by preschian**
- [x] closes https://github.com/kodadot/nft-gallery/issues/5443

### CODE REVIEW

File: ssr-opengraph/src/utils.ts [hunk 1]
```diff
@@ -27,13 +27,19 @@ export const getCollectionById = async (chain: Chains, id: string) => {
...
-  const client = getClient(getChain);
+  const client = clientOf(chain);
   const query = client.itemListByCollectionId(id);
 
   return await client.fetch(query);
 };
 
+export const getItemListByIssuer = async (chain: Chains, id: string) => {
+  const client = clientOf(chain);
+  const query = client.itemListByIssuer(id);
```
Comment (roiLeo) [inline ssr-opengraph/src/utils.ts:38]: No limit? How would it work on big minter account?

**Review by roiLeo — COMMENTED** [review]
I still don't understand why do we need this stack?
Can you explain to me (in a basic way) why is it needed?

It was working before when we used directly `og-image-green-seven` in app

File: ssr-opengraph/src/utils.ts (hunk 1 above)
Comment (roiLeo) [reply ssr-opengraph/src/utils.ts:38]: Yep that what I've seen when playing with it

**Comment by roiLeo** [comment]
Oh! rendering method is why we don't have meta tags in source? I didn't knew about this one.

> client-side rendering vs server-side rendering have their own pros/cons. most of the downside with CSR is about SEO https://nuxt.com/docs/guide/concepts/rendering#cons

Yup, we'll need to rewrite many stuff to use ssr

**Review by yangwao — APPROVED** [review]

**Comment by yangwao** [comment]
@preschian can we get this code for -beta too?

**Pull request merged**